gsplat/cuda/csrc/types.cuh

```diff
@@ -21,4 +24,24 @@ template <typename T> using mat4 = glm::mat<4, 4, T>;
 
 template <typename T> using mat3x2 = glm::mat<3, 2, T>;
 
+
+template <typename T> struct OpType {
+    typedef T type;
+};
+
+template<> struct OpType<__nv_bfloat16> {
```

Comment:
We also need `OpType<float>` and `OpType<double>` for completeness right?

Reply:
Nope! They just use the default `OpType` definition which is the identity map for these types. i.e.

```c++
template <typename T>
struct OpType {
    typedef T type;
};
```